src/ugeneui/src/task_view/TaskStatusBar.cpp

```diff
@@ -34,6 +34,13 @@
 
 namespace U2 {
 
+// Returns a string with the number of notifications, taking into account stack limits.
+static QString notificationCountToString(int notificationCount) {
+    return notificationCount < NotificationStack::MAX_STACK_SIZE
```

Comment:
So you want to show something like ```99+``` instead of ```100+``` here?
I would show '100+' instead.

Reply:
![image](https://user-images.githubusercontent.com/39720063/170412136-462e67e5-6508-4a5a-a8e1-a5474c3a5eb3.png)